Android/app/src/androidTest/java/tie/hackathon/travelguide/Tests/Helpers.java

```diff
@@ -253,8 +240,8 @@ public static void clickOnTheFirstObjectInTheListWhenAmbiguous(String text, int
     }
 
     public static void navigateBackToApp() throws Exception {
-        UiDevice mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
-        mDevice.pressRecentApps();
+        UiDevice Device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
```

Comment:
Daca il avem deja in EspressoTestBase nu trebuie sa-l mai declaram aici. Doar il importam. :)
Numele variabilelor se scriu intotdeauna cu litera mica.